.github/workflows/update-configuration.yml

```diff
@@ -0,0 +1,58 @@
+name: "Update Configuration"
+
+on:
+  workflow_dispatch:
+  push:
+
+jobs:
+  update:
+    name: "Update Configuration in manifest.json"
+    runs-on: ubuntu-latest
+    permissions: write-all
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20.10.0"
+
+      - name: Install deps and run configuration update
+        run: |
+          yarn install --immutable --immutable-cache --check-cache
+          yarn tsc --noCheck --project tsconfig.json
+
+      - name: Update manifest configuration using GitHub Script
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const fs = require('fs');
+            const path = require('path');
+
+            const { pluginSettingsSchema } = require('./src/types/plugin-inputs');
+
+            const manifestPath = path.resolve("${{ github.workspace }}", './manifest.json');
+            const manifest = JSON.parse(fs.readFileSync(manifestPath, 'utf8'));
+
+            const configuration = JSON.stringify(pluginSettingsSchema);
+
+            manifest["configuration"] = JSON.parse(configuration);
+
+            const updatedManifest = JSON.stringify(manifest, null, 2)
+            console.log('Updated manifest:', updatedManifest);
+            fs.writeFileSync(manifestPath, updatedManifest);
+
```

Comment:
If you compile the js with `ncc` I am 90% sure you can skip 

```
          yarn install --immutable --immutable-cache --check-cache
          yarn tsc --noCheck --project tsconfig.json
```

Which will make this much faster.

Reply:
Sure we can do. But this is the configuration generation so not much benefit from making it faster. The `compute` however would be beneficial.